aframe/src/component-anchor.js

```diff
@@ -115,6 +115,7 @@ AFRAME.registerComponent('arjs-anchor', {
                 markerParameters.markersAreaEnabled = false
             }
 
+            markerParameters.minConfidence = _this.data.minCondidence;
```

Comment:
There is a typo error here, ti should be:
```diff
- markerParameters.minConfidence = _this.data.minCondidence;
+ markerParameters.minConfidence = _this.data.minConfidence;
```
i will fix it in the ES6 branch PR #116